content/projects/oop/animals/part2/_index.md

```diff
@@ -67,29 +67,47 @@ Your directory structure should look like this:
     - package.json
 ```
 
-## Instructions
+
+### Python
+
+Your directory structure should look like this:
+
+```
+├── animals
+│   └── animals.py
+├── setup.py
+├── requirements.txt
+├── .gitignore
+└── tests
+    └── test_animals.py
+
+```
+
+
+## General Instructions
+
+1. Update the `Animal` super class `eats()` function to return a String, "Food".
+2. The `Dog` and `Cat` class `sounds()` functions remain as per part 1 of the project.
+3. Make sure the `makeAllSounds()`/`make_all_sounds()` method still works properly.
+
 
 ### For Java
 
 You'll be using JUnit.
-
-1. Create a class called `AnimalTests`.
-2. Update `Animal` super class `eats()` function to return a String, "Food".
-3. Update `Dog` class `sounds()` function to return a `String`, "Bark".
-4. Update `Cat` class `sounds()` function to return a `String`, "Meow".
+Create a class called `AnimalTests`.
 
 ```
 // Java
 
 Dog dog = new Dog()
 
-dog.eats()    // -> 'Food'
-dog.sounds() // -> 'Bark'
+dog.eats()    // returns 'Food'
+dog.sounds() // returns 'Bark'
 
 Cat cat = new Cat()
 
-cat.eats()    // -> 'Food'
-cat.sounds() // -> 'Meow'
+cat.eats()    // returns 'Food'
+cat.sounds() // returns 'Meow'
```

Comment:
since we are not making changes to sound anymore in part 2 (because of the new instructions), should we just remove the sounds() examples and leave the eats() ones or they are still fine there?